src/Chapter2.hs

```diff
@@ -857,7 +895,11 @@ and reverses it.
   function, but in this task, you need to implement it manually. No
   cheating!
 -}
-rewind = error "rewind: Not Implemented!"
+rewind :: [Int] -> [Int]
+rewind [] = []
+rewind (x:xs)
+  | x <= 0 = []
```

Comment:
I see, that your implementation works with lists of integer, and you check that elements of the list shouldn't be negative. I guess, this is not necessary, and you make this function work with any list :slightly_smiling_face: 

```haskell
rewind :: [a] -> [a]
```